src/output-json-mqtt.c

```diff
@@ -112,6 +115,15 @@ static void JsonMQTTLogParseConfig(ConfNode *conf, LogMQTTFileCtx *mqttlog_ctx)
     } else {
         mqttlog_ctx->flags |= MQTT_LOG_PASSWORDS;
     }
+    uint32_t max_msg_log_len = 0;
+    query = ConfNodeLookupChildValue(conf, "msg-log-limit");
```

Comment:
@jasonish what do you think about doing this in rust ? and setting a global variable with this field

Reply:
Probably only worth it if the whole Config function can be moved to Rust.

Reply:
> the whole Config function can be moved to Rust.

Yes, we can, but the way I see it means using a global variable, so unsafe static mut...